PostgreSQL/setup.js

```diff
@@ -0,0 +1,68 @@
+let pgres = require('./index.js');
+
+let createItemTable = `CREATE TABLE items(
+  ID SERIAL PRIMARY KEY NOT NULL,
+  title TEXT NOT NULL,
+  description TEXT,
+  colors TEXT,
+  sizes TEXT,
+  "originalPrice" INT,
+  "salePrice" INT,
+  liked BOOLEAN,
+  "inStock" INT
+)`;
+
+let createReviewsTable = `CREATE TABLE reviews(
+  ID SERIAL PRIMARY KEY NOT NULL,
+  prodId INT REFERENCES items (ID),
+  "easeOfAssembly" INT,
+  "valueForMoney" INT,
+  "productQuality" INT,
+  appearance INT,
+  "worksAsExpected" INT,
+  "overallRating" INT,
+  "createdAt" TEXT,
+  "iRecommendThisProduct" BOOLEAN,
+  header TEXT,
+  body TEXT
+)`;
+pgres.connect();
+
+// the commented code would create a specific database for this module, but then it becomes difficult to immediately connect
+// so simply creating the tables in postgres should be acceptable for now
+
+// pgres.query(`DROP DATABASE IF EXISTS mykea`)
+//   .then(() => {
+//     console.log('creating database');
+//     return pgres.query(`CREATE DATABASE mykea`)
+//   })
+//   .then(() => {
+    pgres.query(`DROP TABLE IF EXISTS reviews`)
```

Comment:
Did you try switching your queries to mykea.query after creating database? From what I recall when I was first getting a sense of postgresql, that worked for me as far as getting the table to the newly create db. But as you said, this  totally works for MVP!

Reply:
I did, and I am able to create and then connect to a database. It just seems a little funky to either manually create the database and then run my scripts, or run the script twice, once to create the database and then error out, and once to actually connect to it. I'll have to look at what you did when I have time.